options/m_option.c

```diff
@@ -51,6 +51,7 @@
 #define OPTION_PATH_SEPARATOR ':'
 #endif
 
+const union m_option_value m_option_value_default;
```

Comment:
Also, strictly speaking, this is no different than doing `{0}` because only the first member will be initialized. I guess it happens to work out due to being put into `.bss` instead of being on the call stack. But maybe we should try to find a proper solution.

ref: http://port70.net/~nsz/c/c99/n1256.html#6.7.8p10

> If an object that has static storage duration is not initialized explicitly, then [...]
> if it is a union, the first named member is initialized

Reply:
It is a proper solution. In practice anything with static storage is initialized fully to zeros, because of how it works.

The standard were also updated to reflect that https://port70.net/~nsz/c/c11/n1570.html#6.7.9p10 and we use C11, so there is nothing to change here.

>  if it is a union, the first named member is initialized (recursively) according to these rules, and any padding is initialized to zero bits;

Reply:
> The standard were also updated to reflect that

I see. In that case it's fine.